core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -111,8 +110,9 @@ class HttpGoogleServicesDAO(
   billingPemEmail: String,
   billingPemFile: String,
   val billingEmail: String,
-  val billingGroupEmail: String,
-  val billingGroupEmailAliases: List[String],
+  billingGroupEmail: String,
+  billingGroupEmailAliases: List[String],
```

Comment:
do we need these billing group things?

Reply:
yes, they're used in the DM PR. i just realised they didn't need to be vals so i un-val'd them